mailpoet/lib/EmailEditor/Integrations/Core/Renderer/Blocks/Button.php

```diff
@@ -2,137 +2,101 @@
 
 namespace MailPoet\EmailEditor\Integrations\Core\Renderer\Blocks;
 
-use MailPoet\EmailEditor\Engine\Renderer\ContentRenderer\BlockRenderer;
 use MailPoet\EmailEditor\Engine\SettingsController;
+use MailPoet\EmailEditor\Integrations\Core\Renderer\Blocks\AbstractBlockRenderer;
```

Comment:
I think this line is redundant because `AbstractBlockRenderer` is from the same namespace. The situation is similar in the `Column` and `Columns` renderer.

Reply:
Resolved in https://github.com/mailpoet/mailpoet/pull/5511/commits/1104d196af9de50211c044f3b09a538d427377ff